wandb/sdk/artifacts/artifact.py

```diff
@@ -1686,6 +1686,64 @@ def download(
         root = root or self._default_root()
         self._add_download_root(root)
 
+        if wandb.run is None:
+            with wandb.init(
+                entity=self._source_entity,
+                project=self._source_project,
+                job_type="auto",
+                settings=wandb.Settings(silent="true"),
+            ):
+                return self._run_artifact_download(
+                    root=root,
+                    recursive=recursive,
+                    allow_missing_references=allow_missing_references,
+                )
```

Comment:
do you want to create a run for the users if there is no one?

Reply:
yes...